sdk/eventhubs/test-resources.json

```diff
@@ -12,9 +12,12 @@
             "type": "string",
             "defaultValue": "javaeventhub"
         },
-        "eventHubNamespaceHostNameSuffix": {
+        "serviceBusEndpointSuffix": {
```

Comment:
Why service bus? This should be Event Hubs still.

Reply:
There is no [eventHubNamespaceHostNameSuffix] parameter record in the configuration of the cloud service, and the Endpoint suffix of the EventHubs Service is exactly the same as that of the Service Bus, so the parameter [serviceBusEndpointSuffix] is used directly.